examples/co2_and_tracer_analysis.py

```diff
@@ -26,7 +28,7 @@ def preprocessing(img: np.ndarray) -> np.ndarray:
     # (which can be created the workflow presented in the Jupyter notebook
     # examples/notebooks/curvature_correction_walkthrough.ipynb).
     curvature_correction = daria.CurvatureCorrection(
-        config_source="./images/config.json", width=2.8, height=1.5
+        config_source=Path("./images/config.json"), width=2.8, height=1.5
```

Comment:
I suddenly got an error here. I don't know. You solved the issue about Path and str yesterday. Is this still following your philosphy?

Reply:
@EStorvik

Reply:
My idea was to only pass strings, and then inside the curvature correction the Path functon would be used.

Reply:
I now realize that you would like to send "Path"s to be read by daria? If so, that is ok with me, but I think that reading with str(Path("...")) inside the functions works equally well. We should choose one convention.

Reply:
I agree. The important thing will be to be sure, that it works both on Linux and Windows.

Users on Windows will use the backslash convention - btw. for each backslash, one needs to use two in Windows...

I will have to read more on pathlib before claiming I know what is going on.

Reply:
I was using windows all yesterday and is seems that Path really does all the job for us. I even tried mixing / and \, and it always worked as long as it was inside Path.

Reply:
I will leave the responsibility for pathlib to you :) Thanks for the effort.